services/accumulo-service/src/main/java/datawave/microservice/audit/config/AuditServiceConfiguration.java

```diff
@@ -0,0 +1,22 @@
+package datawave.microservice.audit.config;
+
+import datawave.microservice.audit.AuditServiceProvider;
+import datawave.microservice.audit.config.discovery.AuditDiscoveryConfiguration;
+import org.springframework.boot.autoconfigure.AutoConfigureAfter;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnProperty(name = "audit.enabled", havingValue = "true")
```

Comment:
Maybe add matchIfMissing to ensure that audit defaults to true?

Reply:
Yeah, good call. Will do